phototo3d.js

```diff
@@ -0,0 +1,220 @@
+/////////////////////////////////////////////////////////////////////
+// Copyright (c) Autodesk, Inc. All rights reserved
+//
+// Permission to use, copy, modify, and distribute this software in
+// object code form for any purpose and without fee is hereby granted,
+// provided that the above copyright notice appears in all copies and
+// that both that copyright notice and the limited warranty and
+// restricted rights notice below appear in all supporting
+// documentation.
+//
+// AUTODESK PROVIDES THIS PROGRAM "AS IS" AND WITH ALL FAULTS.
+// AUTODESK SPECIFICALLY DISCLAIMS ANY IMPLIED WARRANTY OF
+// MERCHANTABILITY OR FITNESS FOR A PARTICULAR USE.  AUTODESK, INC.
+// DOES NOT WARRANT THAT THE OPERATION OF THE PROGRAM WILL BE
+// UNINTERRUPTED OR ERROR FREE.
+/////////////////////////////////////////////////////////////////////
+
+var express	= require('express');
+var bodyParser	= require('body-parser');
+var favicon	= require('serve-favicon');
+var async	= require('async');
+var unirest	= require('unirest');
+var ejs		= require('ejs');
+
+var app = express();
+app.use(bodyParser.json ());
+app.use(express.static (__dirname + '/www'));
+app.use(favicon (__dirname + '/www/images/favicon.ico'));
+app.set('view engine', 'ejs');
+
+var client_id		= 'vbuf6Jx9V4WFPOJ1NPmq0ce0LSk5zDep';
+var client_secret	= 'PZuN7SENlB4QQjO7';

```

Comment:
@eugene-chua - You really shouldn't commit secrets to a public github repository...